<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="usernameLabel">
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="usernameTextBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="passwordLabel">
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="passwordTextBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="showPasswordCheckBox">
     <property name="text">
      <string>Show password</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="messagesLabel">
     <property name="text">
      <string>Messages NOT to show alerts on:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="buttonRemoveSelected">
     <property name="text">
      <string>Remove selected</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QTableWidget" name="tableMessages">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::NoDragDrop</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Message ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Subject</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Sender</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Received</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QCheckBox" name="startWithWindowsCheckBox">
     <property name="text">
      <string>Start with Windows</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonRemoveSelected</sender>
   <signal>clicked()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>removeSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showPasswordCheckBox</sender>
   <signal>stateChanged(int)</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>togglePassword(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>removeSelected()</slot>
  <slot>togglePassword(int)</slot>
 </slots>
</ui>
